deploy-finder-process.md

```diff
@@ -11,31 +11,36 @@ This will enqueue the documents via Sidekiq.
 
 * Once inside backend integration go to: `cd /var/apps/specialist-publisher`
 
-* Check size of the Sidekiq queue using: `Queue.all.first.size`
+* Within `govuk_app_console specialist-publisher` check size of the Sidekiq queue using: `Sidekiq::Queue.all.first.size`
```

Comment:
You just selecting the size of the first queue, but if the app has more than one queue that doesn't tell you the whole thing.
https://github.com/mperham/sidekiq/wiki/API#stats